contents/docs/integrations/microsoft-teams.md

```diff
@@ -12,18 +12,21 @@ showTitle: true
 1. The dialog window will present a unique URL that will map to the channel. Make sure that you copy and save the URL—you will need to provide it to the outside service.
 1. Select the Done button. The webhook will be available in the team channel.
 
-[See Microsoft Team's documentation for more info](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook#add-an-incoming-webhook-to-a-teams-channel)
+[See the Microsoft Teams Documentation for more info](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook#add-an-incoming-webhook-to-a-teams-channel).
```

Comment:
I think "documentation" could be lowercase, it's not the name of a product or anything

Reply:
The reason I have it as upper case is because I see it as the title of a page, you see? But yeah lowercase should be fine